apps/communications/contacts/style/contacts.css

```diff
@@ -1118,3 +1156,260 @@ html[dir="rtl"] #custom-tag-reset {
 html[dir="rtl"] .force-align[dir="auto"] {
   text-align: right;
 }
+
+#order-by-lastname{
+  padding: 0;
+  border: none;
+}
+
+[data-type="list"] .contact-item[data-visited="false"] > aside > span {
+    display: block;
+}
+
+li.ice-hide-contact{
+  display: none;
+}
+
+[data-type="list"] header {
+  display: none;
+}
+
+#ice-description {
+  font-size: 1.6rem;
+  margin: -0.5rem;
+  text-align: left;
+  padding: 1rem 0.8rem;
+}
+
+#ice-settings li button:before {
+  display: none;
+}
+
+form[role="dialog"][data-type="confirm"]#loading-overlay {
+  background: #868686;
+}
+
+.ice-image {
+  height: 5rem;
+  display: block;
+  width: 5rem;
+  border: 0.1rem solid #ffffff;
+  border-radius: 2.5rem;
+  float: left;
+  overflow: hidden;
+  background-position: left center;
+  image-rendering: auto;
+  position: relative;
+  background: transparent none repeat scroll center center / cover;
+}
+
+.ice-default-image{
+  background: transparent url("images/Imagery.png") no-repeat scroll 50% 50%/7rem;
+}
+
+.ice-default-image:before {
+  content: attr(data-group);
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  color: #ffffff;
+  font-size: 2.2rem;
+  font-style: italic;
+  line-height: 5rem;
+  text-align: center;
+  text-transform: uppercase;
+}
+
+#select-ice-contact-1 {
+  padding: 0 0 0 1.5rem;
+}
+
+#select-ice-contact-2 {
+  padding: 0 0 0 1.5rem;
+}
+
+#confirmation-message {
+  background-color: rgba(0, 0, 0, 0.7);
+}
+
+#confirmation-message section h1 {
+  padding-left: 0.3rem;
+  margin-top: 1rem;
+}
+
+#confirmation-message section p {
+  padding: 0.3rem;
+  border-top: 0.1rem solid #ffffff;
+}
+
+#settingsFb {
+  font-style: italic;
+}
+
+#fb-aside {
+  font-style: normal;
+}
+
+#search-abc {
+  font-size: 1.5rem;
+  color: #cdd5e2;
+  font-style: italic;
+  line-height: 3rem;
+}
+
+#search-image {
+  background-image: url("images/search.svg");
```

Comment:
We don't have any existing .svg images in the app. I doubt we want to start with that now.  There are a lot of svg icons embedded in our icon font, however, and maybe there is already something there that you can use.  I don't have the details myself, unfortunately.